docs/topics/proc_deploying-the-circuit-breaker-booster-using-the-oc-cli-client.adoc

```diff
@@ -9,11 +9,15 @@
 
 .Prerequisites
 
-* The booster application created using the {launcher} application.
+ifndef::parameter-openshiftlocal[]
+* The booster application created using link:{link-launcher-oso}[{launcher-oso}].
 For more information, see xref:deploying-the-booster-using-launcher_{context}[].
+endif::[]
 
 ifdef::parameter-openshiftlocal[]
-* Your {launcher} URL.
+* The booster application created using the {launcher} service.
+For more information, see xref:deploying-the-booster-using-launcher_{context}[].
+* Your {launcher} service URL.
```

Comment:
This will be the only conditionalized item in the list then. You can write:

```
ifdef::parameter-openshiftlocal[* Your {launcher} service URL.]
```

for brevity.